test/unit/test_loss_image.py

```diff
@@ -118,3 +118,33 @@ def test_get_config(self):
             name="LocalNormalizedCrossCorrelation",
         )
         assert got == expected
+
+
+class TestGlobalNormalizedCrossCorrelation:
+    @pytest.mark.parametrize(
+        "y_true,y_pred,shape,expected",
+        [
+            (0.6, 0.3, (3,), 1),
+            (0.6, 0.3, (3, 3), 1),
+            (0.6, -0.3, (3, 3), 1),
+            (0.6, 0.3, (3, 3, 3), 1),
+        ],
+    )
+    def test_output(self, y_true, y_pred, shape, expected):
+
+        y_true = y_true * np.zeros(shape=shape)
```

Comment:
What's the point of this op, if we do y_true * np.zeros == np.zeros? IE what is the point of reassigning to np.zeros from pytest parametrize? Not sure if this is a bug. Can you explain further?

Reply:
Yes sorry, my miss.